# start
geometry_msgs/PoseStamped start

int8 planning_mode
float64 allocated_planning_time
bool return_first_solution

# added for full body experiments without gazebo
float64[] rarm_start
float64[] larm_start
float64[] body_start
float64[] rarm_goal
float64[] larm_goal
float64[] body_goal

# goal
geometry_msgs/PoseStamped goal

# tolerance
float64 xyz_tolerance
float64 roll_tolerance
float64 pitch_tolerance
float64 yaw_tolerance

# right arm position on the object (in the object frame)
geometry_msgs/PoseStamped rarm_object

# left arm position on the object (in the object frame)
geometry_msgs/PoseStamped larm_object

bool use_ompl
bool use_kd_heuristic

float64 initial_eps
float64 final_eps
float64 dec_eps

float64 egraph_eps
float64 final_egraph_eps
float64 dec_egraph_eps

bool use_egraph
bool feedback_path
bool save_egraph

---

# two arm trajectory
trajectory_msgs/JointTrajectory trajectory
trajectory_msgs/JointTrajectory body_trajectory

# old planning stats
string[] stats_field_names
float64[] stats

# new planning stats
string[] stat_names
float64[] stat_values
